controllers/filterable-list.js

```diff
@@ -1,8 +1,6 @@
-var edit = require('../services/edit'),
-  dom = require('@nymag/dom'),
+const dom = require('@nymag/dom'),
```

Comment:
Yep, we're slowly moving towards using const for these, [per Byron's comment here](https://github.com/nymag/clay-kiln/pull/555#discussion_r73737926)